### PR TITLE
Added list-plans (sub)command.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore-client/187/

**Review by mattyw — APPROVED** [review]

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore-client/192/

File: dependencies.tsv [hunk 1]
```diff
@@ -1,6 +1,5 @@
 github.com/ajstarks/svgo	git	89e3ac64b5b3e403a5e7c35ea4f98d45db7b4518	2014-10-04T21:11:59Z
 github.com/beorn7/perks	git	3ac7bf7a47d159a033b107610db8a1b6575507a4	2016-02-29T21:34:45Z
-github.com/gabriel-samfira/sys	git	9ddc60d56b511544223adecea68da1e4f2153beb	2015-06-08T13:21:19Z
```
Comment (rogpeppe) [inline dependencies.tsv]: This isn't right. When I run godeps on charmstore-client master, I don't see any change. I think you might be running an old version of godeps.

**Review by rogpeppe — APPROVED** [review]
LGTM modulo one issue, thanks.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore-client/193/

**Comment by alesstimec** [comment]
$$merge$$

**Comment by alesstimec** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-client-merge

**Pull request merged**